Makefile

```
@@ -174,13 +174,20 @@ LICENSE_IGNORES = \
 	-ignore 'dist/**' \
 	-ignore 'vendor/**' \
 	-ignore '**/testdata/**' \
+	-ignore 'recipes/evidence/**/*.yaml' \
 	-ignore 'THIRD_PARTY_NOTICES.md' \
 	-ignore '.licenses-cache/**'
 
 .PHONY: license
 license: ## Add/verify license headers in source files
```

checkmake (0.3.2)

[warning] Target body for "license" exceeds allowed length of 5 lines (8).

(maxbodylength)